src/docbkx/chapter12-getting-started-desktop.xml

```diff
@@ -358,10 +358,10 @@
 			</mediaobject>
 		</section>
 		<tip>
-			<para>
-				In the bottom part of the canvas, a help message is displayed
-				instructing what needs to be done in current moment to build a valid
-				job.
+			<para>				
+                 The canvas displays messages (in the bottom of the screen) which
```

Comment:
It's a little weirdly formatted, but no one will notice in the sea of weird formatting called "our documentation XMLs" anyway ;)